--- node_modules/java/src/utils.cpp	2021-12-28 15:18:48.000000000 +0800
+++ node_modules/java/src/utils.fix.cpp	2021-12-28 15:22:30.000000000 +0800
@@ -484,7 +484,8 @@
 #if NODE_VERSION_AT_LEAST(13, 0, 0)
   v8::Local<v8::ArrayBuffer> newArrayBuffer(void* elems, size_t length) {
     v8::Local<v8::ArrayBuffer> ab = v8::ArrayBuffer::New(v8::Isolate::GetCurrent(), length);
-    memcpy(ab->GetBackingStore()->Data(), elems, length);
+    // memcpy(ab->GetBackingStore()->Data(), elems, length);
+    auto arr = v8::Uint8Array::New(ab, 0, length);
     return ab;
   }
 #elif NODE_VERSION_AT_LEAST(4, 0, 0)
